doc/source/reference/random/bit_generators/index.rst

```diff
@@ -12,8 +14,68 @@ setting the state, jumping or advancing the state, and for accessing
+  the same, but with a different state and so it is not possible to produce a
+  sequence identical to MT19937. See the `dSFMT authors' page`_.
+* `Xoshiro256 <xoshiro256>` and `Xoshiro512 <xoshiro512>` - The most recently
+  introduced XOR, shift, and rotate generator. More information about these bit
+  generators is
+  available at the `xorshift, xoroshiro and xoshiro authors' page`_.
+* `ThreeFry <threefry>` and `Philox <philox>` - counter-based generators
+  capable of being advanced an arbitrary number of steps or generating
+  independent streams. See the `Random123`_ page for more details about this
+  class of bit generators.
+* `PCG32 <pcg32>` and `PCG64 <pcg64>` are permutation-congruential generators.
+  More information is available on the `PCG authors' page`_.
+
+.. _`dSFMT authors' page`: http://www.math.sci.hiroshima-u.ac.jp/~m-mat/MT/SFMT/
+.. _`PCG author's page`: http://www.pcg-random.org/
+.. _`xorshift, xoroshiro and xoshiro authors' page`:  http://xoroshiro.di.unimi.it/
+.. _`Random123`: https://www.deshawresearch.com/resources_random123.html
+
+Summary:
+
+============== ==================== ================ ======== ======= =====================
+BitGenerator   period               jumped distance  speed relative
+                                                     to dSFMT [1]_
+-------------- -------------------- ---------------- ---------------- ---------------------
+                                                     linux64  win32
+============== ==================== ================ ======== ======= =====================
+``MT19937``                          :math:`2^{128}`  0.5x     5x     Also used in python's
+                                                                      `random`
+``dSFMT``                            :math:`2^{128}`  1x       10x    Faster version of MT19937
+``Xoshiro256`` :math:`2^{256} - 1`  :math:`2^{128}`
+``Xoshiro256`` :math:`2^{1024} - 1` :math:`2^{512}`
+``ThreeFry``   :math:`2^{256} - 1`  :math:`2^{128}`                   counter-based
+``PHilox``     :math:`2^{256} - 1`  :math:`2^{128}`                   counter-based
+``PCG32``      :math:`2^{64}`       :math:`2^{32}`                     
+``PCG64``      :math:`2^{128}`      :math:`2^{64}`                     
+============== ==================== ================ ======== ======= =====================
+
+.. rubric:: Footnotes
+
+.. [1] More is better. As always with benchmarks, these are rough guides, your
+       experience may vary. Win32 is relative to linux64, ``dSFMT``
+
+
+Recommendations
+===============
+The recommended generator for single use is :class:`~.xoshiro256.Xoshiro256`.
+The recommended generator for use in large-scale parallel applications is
+:class:`~.xoshiro512.Xoshiro512` where the `~.xoshiro512.Xoshiro512.jumped`
+method is used to advance the state. For very large scale applications --
+requiring 1,000+ independent streams, :class:`~pcg64.PCG64` or
+:class:`~.philox.Philox` are the best choices.
```

Comment:
Should we mention here that MT algorithms fail statistical quality tests? What would be a good phrasing?

Reply:
This block is out of date given the recent discussions and should be uspdated.  Ultimately it wil ldepend on what generators are included.